__tests__/users/App.test.js

```diff
@@ -89,4 +89,53 @@ describe('App Component', () => {
     const url = await page.url();
     expect(url).toContain('?tab=verified');
   });
+
+  it('should update the URL query string on search', async () => {
+    const initialUrl = await page.url();
+
+    await page.waitForSelector('.search_field');
+
+    await page.type('.search_field', 'John Doe');
+    await page.click('.search_button');
+
+    const updatedUrl = await page.url();
+
+    expect(updatedUrl).not.toEqual(initialUrl);
```

Comment:
instead of chacking for inequality, please check for the correctness
If the string got appended or not in the URL

I the current case if the URL becomes, realdevsquad.com also then tests will pass

Reply:
fixed and pushed commit for this.